src/style.css

```diff
@@ -1,3 +1,62 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  list-style: none;
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them. It's acceptable to use `box-sizing: border-box;` globally, but other styles like `margin`, `padding`, and `list-style` should be set on specific elements.